shared/src/main/scala/io/kaitai/struct/translators/GoTranslator.scala

```diff
@@ -469,6 +471,35 @@ class GoTranslator(out: StringLanguageOutputWriter, provider: TypeProvider, impo
     ResultLocalVar(v)
   }
 
+  def trInterpolatedStringLiteral(exprs: Seq[Ast.expr]): TranslatorResult = {
+
+    exprs match {
```

Comment:
I don't like the extra newline here, but that's absolutely minor:

```suggestion
  def trInterpolatedStringLiteral(exprs: Seq[Ast.expr]): TranslatorResult = {
    exprs match {
```

Reply:
That's clearly not intentional, thanks for catching this!